modules/ROOT/pages/global-elements.adoc

```diff
@@ -0,0 +1,65 @@
+= Global Elements
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: studio, server, components, connectors, elements, palette, global elements, configuration elements
+
+A Global Element is a reusable object containing parameters that any number of elements in a flow can share. You reference a global element from Anypoint Connectors or components in your Mule application.
+
+Use the global element to apply configuration details to multiple local elements in flows. Create one global element that defines parameters and configuration details, and then reference the global element from any flow element that uses this configuration. This practice enables you to ensure consistency across flow elements.
+
+Note that some components require their properties stored in a global element.
+
+== Prerequisites
```

Comment:
Prerequisites to what? Understanding GEs, or creating GEs? 
I think we typically use this title in the context of a Task, which doesn't show up here until later.